### PR TITLE
Rename build to build.sh

### PR DESCRIPTION
**Pull request by janisozaur**
`build` is commonly used directory for setting up project and compiling inside of it.

`build.sh` clearly denotes this is a shell script.

### CODE REVIEW

**Comment by janisozaur** [comment]
I haven't got a clue why the Windwos[shared] failed, @IntelOrca can you check?

File: CMakeLists.txt [hunk 1]
```diff
@@ -104,12 +104,16 @@ endif ()
 
 if (ENABLE_LIBPNG)
     if (MSVC)
-        find_package(libpng CONFIG REQUIRED)
+        find_package(PNG CONFIG REQUIRED)
```
Comment (IntelOrca) [inline CMakeLists.txt:107]: Perhaps due to this change?

Comment (janisozaur) [reply CMakeLists.txt:107]: Now it is, but it started before it was introduced. Just rebuilding the project would probably trigger it again

Comment (IntelOrca) [reply CMakeLists.txt:107]: I don't know then, I haven't maintained this for a long time. It is possible something has changed with how you include the package.

Comment (janisozaur) [inline CMakeLists.txt:107]:
```suggestion
        find_package(PNG REQUIRED)
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -103,13 +103,19 @@ if (ENABLE_DISCORD_RPC)
...
+    if (NOT MSVC)
+        find_package(PNG REQUIRED)
     endif ()
     target_compile_definitions(sawyer PRIVATE CS_ENABLE_LIBPNG)
 
-    if (${VCPKG_TARGET_TRIPLET} MATCHES "-static$")
-        merge_static_libraries(sawyer $<TARGET_FILE:png_static>)
+    if (MSVC)
+        if (${VCPKG_TARGET_TRIPLET} MATCHES "-static$")
+            find_package(PNG CONFIG REQUIRED)
```
Comment (janisozaur) [inline CMakeLists.txt:113]:
```suggestion
            find_package(libpng CONFIG REQUIRED)
```

**Comment by janisozaur** [comment]
All green now. Can you take a look and merge?

**Pull request merged**